cves/kernel/CVE-2017-0861.yml

```diff
@@ -450,7 +457,10 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    The mistake that caused this vulnerability was not paying attention to when a pointer is freed
+    and that it can be used again by another thread (could be any thread).  This is seems like something that's hyper-difficult to
+    catch and I do not expect a mortal human to be able to spot this vulnerability by looking at the code alone.
```

Comment:
I do find this explanation to be funny, but would it be possible to clarify about how you think this vulnerability was introduced in the first place (lapse of judgement, poor planning, etc.)?

Reply:
I would say this vulnerability was simply an unfortunate by product of solving a problem using software, and would be harsh to call it lapse of judgement or poor planning.